neoexchange/core/models/body.py

```diff
@@ -149,6 +159,8 @@ class Body(models.Model):
     updated             = models.BooleanField('Has this object been updated?', default=False)
     ingest              = models.DateTimeField(default=datetime.utcnow, db_index=True)
     update_time         = models.DateTimeField(blank=True, null=True, db_index=True)
+    analysis_status     = models.CharField('Current Analysis Status', max_length=2, choices=STATUS_CHOICES, db_index=True, default='0')
```

Comment:
Similarly here, I would make this a `SmallIntegerField` and `int`ify the `STATUS_CHOICES`

Reply:
This is an easy one since it's not based on other people's stuff.